### ISSUE TITLE
[Bug]: ActionButton cannot activate longpress content using JAWS

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#3197]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-action-button

### Expected behavior

When longpress content is present I should be able to activate that content using JAWS. 

### Actual behavior

Cannot activate the longpress content using JAWS

### What browsers are you seeing the problem in?

Firefox, Chrome, Microsoft Edge

### How can we reproduce this issue?

1. Start JAWS
1. Go to https://opensource.adobe.com/spectrum-web-components/components/action-button/#action-button-with-hold-affordance
2. Focus on one of the buttons with longpress using the tab key
3. Try using space or alt+down arrow
4. Note that the longpress content does not appear

This also reproduces with NVDA except here alt+down arrow does show the longpress content.

**Comment by najikahalsema** [comment]
@jnurthen, has the license for JAWS gotten attached to Assistiv Labs?

**Comment by jnurthen** [comment]
Yes it has.

**Comment by caseyisonit** [comment]
Hello @jnurthen in order to help us triage your issue, [can you please review our bug severity classification guide](https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md#issue-severity-classification) and provide us with the appropriate severity classification? Please update the label to the correct severity level and remove `SEV missing` label.

If we don't hear from you in 2 weeks, we will close this issue as not planned.

<img width="1024" height="1028" alt="Image" src="https://github.com/user-attachments/assets/d5fd78ee-6e8e-4967-97f3-4e4198b79f13" />
